pkg/labels/labels.go

```diff
@@ -7,4 +7,10 @@ const (
 	MasterRole = "node-role.kubernetes.io/master"
 	// ControlPlaneRole is the new role label of control plane nodes
 	ControlPlaneRole = "node-role.kubernetes.io/control-plane"
+	// ProcessingConditionType is the condition type used to signal the remediation has started and it is in progress, or has finished
+	ProcessingConditionType = "Processing"
+	// SucceededConditionType is the condition type used to signal whether the remediation was successful or not
+	SucceededConditionType = "Succeeded"
+	// NhcTimeOutAnnotation is the annotation set by NHC to signal the operator that it surpassed its timeout and shall stop its remediation
+	NhcTimeOutAnnotation = "remediation.medik8s.io/nhc-timed-out"
```

Comment:
1.  think it would be better to put in another pkg (i.e annotations) 
2. Assuming we do so get rid of the "Annotation" suffix in the var name